python/pypkgconf/callbacks.py

```diff
@@ -0,0 +1,105 @@
+from ._libpkgconf import ffi, lib
+
+from .constants import Flags
+from .logger import logger
+
+
+@ffi.def_extern()
+def error_handler(msg: str, client, data) -> bool:
```

Comment:
It would be a great idea to add types to to everything

Reply:
yes. It is just not super-obvious with the cffi bindings. But I could try... But on another hand, I don't want to spend too much time on this PR if it is not to be adopted.

Reply:
Sure, let's wait and see what the maintainer says.